src/pkg/backup/backup.go

```diff
@@ -49,9 +49,6 @@ type Backup struct {
 	// stats are embedded so that the values appear as top-level properties
 	stats.ReadWrites
 	stats.StartAndEndTime
-
-	// Deprecated
-	stats.Errs // replaced with fault.Errors
```

Comment:
I'm assuming this doesn't cause unmarshal of older serialized backup models to fail. I think the default behavior is that missing fields are just ignored

Reply:
Yup, shouldn't cause any serialization errors.  Worst case is that old backups lose the count on `Completed (X errors)`